{
  "id": "PYSEC-2019-180",
  "modified": "2023-11-08T04:01:38.735616Z",
  "published": "2019-03-21T21:29:00Z",
  "aliases": [
    "CVE-2019-7539",
    "GHSA-gcm4-q2pg-xw89"
  ],
  "details": "A code injection issue was discovered in ipycache through 2016-05-31.",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "ipycache",
        "purl": "pkg:pypi/ipycache"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            }
          ]
        }
      ],
      "versions": [
        "0.1.0",
        "0.1.1",
        "0.1.2",
        "0.1.3",
        "0.1.4"
      ],
      "database_specific": {
        "source": "https://github.com/pypa/advisory-database/blob/main/vulns/ipycache/PYSEC-2019-180.yaml"
      }
    }
  ],
  "references": [
    {
      "type": "REPORT",
      "url": "https://github.com/rossant/ipycache/issues/47"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/advisories/GHSA-gcm4-q2pg-xw89"
    }
  ]
}